lib/ansible/modules/cloud/amazon/ec2_vpc_net_facts.py

```diff
@@ -121,7 +121,9 @@ def main():
     if region:
         try:
             connection = connect_to_aws(boto.vpc, region, **aws_connect_params)
-        except (boto.exception.NoAuthHandlerFound, StandardError) as e:
+        except (StopIteration, GeneratorExit, KeyboardInterrupt, SystemExit):
```

Comment:
Thanks for making this Python3 compatible. You can remove this line and the line below. Catching boto.exception.NoAuthHandlerFound and Exception should be fine.